features/shop/promotion/receiving_percentage_discount_promotion_on_order.feature

```diff
@@ -0,0 +1,27 @@
+@promotion
+Feature: Receiving percentage discount promotion on order
+    In order to pay proper amount while buying promoted goods
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store is operating on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And there is a promotion "Holiday promotion"
+        And it gives "20%" percentage discount to every order
+
+    @todo
+    Scenario: Receiving percentage discount for my cart
+        When I add product "PHP T-Shirt" to the cart
+        Then my cart total should be "€80.00"
+        And my cart promotions should be "-€20.00"
+
+    @todo
+    Scenario: Receiving percentage discount does not affect the shipping fee
+        Given the store has "DHL" shipping method with "€10.00" fee
+        And I am logged in customer
```

Comment:
To proceed order we should be logged in or proceed logging in or proceed as guest. However, I think that being logged in as a customer is the most basic case and it should be used here.